.changeset/four-mice-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Update control-sm & control-lg font size variables
```

Comment:
Super nit: missed the period sign at the end of the sentence.

Reply:
👍updated